apache/zeppelin
# [ZEPPELIN-4514]. Job URL link is missing when refreshing note

### What is this PR for?

This PR use gson's ExclusionStrategy to change the serialization of Paragraph dynamically: serialize runtimeInfos for frontend but not for note file. 


### What type of PR is it?
[ Improvement ]

### Todos

### What is the Jira issue?
* https://issues.apache.org/jira/browse/ZEPPELIN-4514

### How should this be tested?
* Verify it manually, run spark job at one note. Then open another tab to open the note to verify that the job url link is still there.

### Screenshots (if appropriate)

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No
